samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -613,6 +732,23 @@ static void radio_rx(uint8_t mode, uint8_t channel, enum transmit_pattern patter
 	radio_disable();
 
 	radio_mode_set(NRF_RADIO, mode);
+
+#if defined(CONFIG_SOC_SERIES_NRF54HX)
+#ifdef NRF_TRUSTZONE_NONSECURE
+	uint32_t base = 0x4302C000;
+#else
+	uint32_t base = 0x5302C000;
+#endif /* ifdef NRF_TRUSTZONE_NONSECURE */
+	if (mode == NRF_RADIO_MODE_IEEE802154_250KBIT) {
+		uint32_t temp;
```

Comment:
Not sure what definitions will be in MDK for this repository but we could consider using register and bitmask definition instead of magic numbers